insalan/cms/models.py

```diff
@@ -8,25 +8,37 @@ def constant_definition_validator(content: str):
     """
     validator to ensure that any used constant in content is defined
     """
-    regex = re.compile('\$\{(?P<name>[^\{\}]*)\}')
-    constant_list = set(re.findall(regex, content)) #get the constant names in the content
-    p_constants =  set(constant.name for constant in Constant.objects.all()) #retrieve every constant names
+    regex = re.compile("\$\{(?P<name>[^\{\}]*)\}")
+    constant_list = set(
+        re.findall(regex, content)
+    )  # get the constant names in the content
+    p_constants = set(
+        constant.name for constant in Constant.objects.all()
+    )  # retrieve every constant names
 
-    excess_constants = constant_list - p_constants.intersection(constant_list) # Get the constants in the content but not defined
+    excess_constants = constant_list - p_constants.intersection(
+        constant_list
+    )  # Get the constants in the content but not defined
 
     if len(excess_constants) > 0:
         raise ValidationError(
-                _(f"Des constantes non définies sont utilisées: {', '.join(sorted(excess_constants))}"))
+            _(
+                f"Des constantes non définies sont utilisées: {', '.join(sorted(excess_constants))}"
```

Comment:
This kind of messes up the translation string if I remember correctly, but I'm just commenting on it here to make a mental note to go check later

Reply:
what's the best way to deal with this kind of strings ?

Reply:
changed to format() in [240d910](https://github.com/InsaLan/backend-insalan.fr/pull/92/commits/240d910b0ca06445987028748b3ac0c3586174e2)